js-1-4/js1-4.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const title = 
+`===========================
+  現在持っているのタスク一覧
+===========================`;
+const tasks = [
+  { item: '机を片付ける', genre: '掃除'},
+  { item: '牛乳を買う', genre: '買い物'},
+  { item: '散歩する', genre: '運動'},
+];
+
+function answer () {  
+  console.log(title); 
+  Object.keys(tasks).forEach((value, index) => {
```

Comment:
`forEach` を使うほうが簡便かと思います。

```suggestion
  tasks.forEach((value, index) => {
```

Reply:
ありがとうございます。
修正しました。
どちらが良いか判断できず、両方記載してみました。